R/pkg/R/DataFrame.R

```diff
@@ -2121,7 +2121,7 @@ setMethod("arrange",
           })
 
 #' @rdname arrange
-#' @name orderBy
```

Comment:
I deleted this line eventually - otherwise it would have duplicated alias for orderBy.